js/coinex.js

```diff
@@ -246,10 +262,34 @@ module.exports = class coinex extends Exchange {
         //                 "trading_name": "WAVES",
         //                 "trading_decimal": 8
         //             }
+        //             ...
         //         }
         //     }
         //
-        const markets = this.safeValue (response, 'data', {});
+        //     privateGetMarginConfig
```

Comment:
As this call is private and the first one is private I'm thinking it may be best to be behind some kind of option?

Reply:
We can check if the exchange instance has the necessary credentials and only issue that call, if those credentials are available, see how it's done here, for example: https://github.com/ccxt/ccxt/blob/master/js/binance.js#L1224